.travis.yml

```diff
@@ -22,7 +21,3 @@ matrix:
 script:
   - git fetch --unshallow
   - ./build.sh
-notifications:
```

Comment:
Ah so this can be removed completely now?

Reply:
Well, I'm sort of testing this out. I think these settings relate to email notifications, rather than GitHub build statuses and we don't use the email notifications anyway as we have everything we need on the PR. So I'm hopeful this can be removed yeah.

Reply:
Seems like it's reported the status fine without them.